cmd/build/helmify/replacements.go

```diff
@@ -47,6 +47,8 @@ var replacements = map[string]string{
 
 	`HELMSUBST_DEPLOYMENT_AUDIT_IMAGE_PULL_SECRETS: ""`: `{{- toYaml .Values.image.pullSecrets | nindent 8 }}`,
 
+	`HELMSUBST_DEPLOYMENT_AUDIT_EXTRAENV: ""`: `{{- toYaml .Values.audit.extraEnv | nindent 8 }}`,
```

Comment:
this should be `HELMSUBST_DEPLOYMENT_AUDIT_MANAGER_EXTRAENV`

Reply:
can we use `HELMSUBST_DEPLOYMENT_AUDIT_..` (no manager) for consistency?

Reply:
Thank you for the specific problem description. I've updated the PR to fix the mismatch. I agree with @sozercan, and renamed `HELMSUBST_DEPLOYMENT_AUDIT_MANAGER_EXTRAENV` to `HELMSUBST_DEPLOYMENT_AUDIT_EXTRAENV` in `./cmd/build/helmify/kustomize-for-helm.yaml` for consistency.